rust-src/wallet_library/Cargo.toml

```diff
@@ -13,6 +13,8 @@ serde_json = "1.0"
 thiserror = "1.0"
 either = "1.6"
 ed25519-dalek = "2.0"
+chrono = "0.4.24" # the patch version to match base
```

Comment:
Why do you need the patch version here? Base already sets it, so this won't really do anything.

Reply:
Ahh, I thought it might use the latest here if I didn't specify. I've removed the patch.